client/styles/components/_github-button.scss

```diff
@@ -12,14 +12,19 @@
       border-color: getThemifyVariable('secondary-text-color');
     }
   }
-  width: #{300 / $base-font-size}rem;
+  width:95%;
```

Comment:
rather than doing a percent width here, i think it makes sense to add a fixed padding or margin.